.github/workflows/e2e-tests-ci.yml

```diff
@@ -57,3 +57,22 @@ jobs:
       - name: ci/playwright/npm-check
         run: |
           npm run check
+  smoketests:
+    runs-on: ubuntu-22.04
+    needs:
+      - cypress-check
+      - playwright-check
+    defaults:
+      run:
+        working-directory: e2e-tests
+    steps:
+      - name: ci/checkout-repo
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      - name: ci/e2e-smoketests
+        run: |
+          make
+          # Assert that the run contained 0 failures
+          CYPRESS_FAILURES=$(find ../cypress/results -name '*.json' | xargs -l jq -r '.stats.failures' | jq -s add)
```

Comment:
This is failing because the folder does not exist . Probably a path issue.

Reply:
Fixed, thanks!